internal/errors/resolver/update.go

```diff
@@ -0,0 +1,66 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package resolver
+
+import (
+	"github.com/GoogleContainerTools/kpt/internal/errors"
+	"github.com/GoogleContainerTools/kpt/internal/util/update"
+)
+
+//nolint:gochecknoinits
+func init() {
+	AddErrorResolver(&updateErrorResolver{})
+}
+
+var (
+	//nolint:lll
+	pkgNotGitRepo = `
+Package {{ printf "%q" .repo }} is not within a git repository. Please initialize a repository using 'git init' and then commit the changes using 'git commit -a -m "<commit message>"'.
```

Comment:
should we just leave it at using `git commit`. Since `git commit -a` won't always work if there are new files (Need to use `git add`). I rather have this more general and correct than more specific and problematic.